public/translation_fr.json

```diff
@@ -14,19 +14,19 @@
         "level": "NIVEAU",
         "downtime": "TEMPS LIBRE",
         "status": "VISIBILITÉ",
-        "delete-message": "Êtes-vous certian de vouloir supprimer ce/ces personnage(s)",
+        "delete-message": "Êtes-vous certain de vouloir supprimer ce/ces personnage(s)?",
         "edit-character": "Modifier le personnage",
-        "edit-entry": "Modifier le saisie",
-        "fill-out-fields": "Veuillez remplier les champs suivants avec les détails de votre personnage",
+        "edit-entry": "Modifier la session",
+        "fill-out-fields": "Veuillez remplier les champs suivants avec les détails de votre personnage.",
         "public-status": "Acceptez-vous que ce personnage soit rendu publique?",
```

Comment:
I'm pretty sure `personnage` is a `masculin` word. Therefore `publique` should be replaced with `public`. [Proof](https://www.larousse.fr/dictionnaires/francais/personnage/59805)